nextflow.config

```
@@ -291,57 +304,59 @@ params {
 
    skipVersionCheck = false
 
-   emgbAnnotatedGenes_image = "quay.io/emgb/annotatedgenes2json:2.5.2z"
-   emgbAnnotatedContigs_image = "quay.io/emgb/annotatedcontigs2json:2.2.4z"
-   emgbAnnotatedBins_image = "quay.io/emgb/annotatedbins2json:2.4.0z"
-   pysradb_image = "quay.io/biocontainers/pysradb:1.4.1--pyhdfd78af_0"
-   minimap2_image= "quay.io/biocontainers/minimap2:2.24--h7132678_1"
-   metaflye_image = "quay.io/biocontainers/flye:2.9--py36h7281c5b_1"
-   metacoag_image = "quay.io/metagenomics/toolkit-metacoag:1.0.0-0"
-   porechop_image = "quay.io/biocontainers/porechop:0.2.4--py39hc16433a_3"
-   bandage_image = "quay.io/biocontainers/bandage:0.8.1--hc9558a2_2"
-   nonpareil_image = "quay.io/biocontainers/nonpareil:3.4.1--r41h9f5acd7_1"
-   ubuntu_image = "quay.io/metagenomics/toolkit-ubuntu:20.04-0"
-   carveme_image = "quay.io/metagenomics/toolkit-carveme:1.6.1-0"
-   cobra_image = "quay.io/biocontainers/cobra:0.21.0--pyh5e36f6f_1"
-   memote_image = "quay.io/metagenomics/toolkit-memote:0.13.0-0"
+   apptainer = params.containsKey("apptainer") && params.apptainer == true ? true : false
```

```suggestion
   apptainer = params.containsKey("apptainer") && params.apptainer == true
```

Only a suggestion, a tiny bit shorter and should also work

---

I even tried to an even shorter expression `params.containsKey("apptainer") && params.apptainer` and it did not work.